packages/react-devtools-shared/src/devtools/views/Components/SelectedElement.js

```diff
@@ -362,6 +363,13 @@ function InspectedElementView({
         });
       }
     };
+  } else if (type === ElementTypeContext) {
```

Comment:
It's not clear if this should be added to the previous branch that is entered for function/memo/forwardRef components or if this would be the wrong abstraction.

Reply:
I think this was the right call: https://github.com/facebook/react/pull/18257/files#diff-fc6493b7eb53b9380e69716f370968a7R367

Reply:
Not sure. That one is because it was a separate concept for legacy. But this one is literally editing props.